### PR TITLE
Delete notes

### PR DESCRIPTION
**Pull request by jagruti-gothwal**

### CODE REVIEW

**Review by pr656d — CHANGES_REQUESTED** [review]
Resolve conflicts

File: public/js/notes.js [hunk 1]
```diff
@@ -4,6 +4,16 @@ function update() {
     let str = "";
     if (localStorage.getItem('itemsJson') != null) {
         itemsJson = JSON.parse(localStorage.getItem('itemsJson'));
+
+        itemsJson.forEach((element, index) => {
+            str += `
+    <tr>
```
Comment (pr656d) [inline public/js/notes.js]: Indentation issue

File: templates/views/notes.hbs [hunk 2]
```diff
@@ -32,5 +32,9 @@
         </section>
     {{>footer}}
 <script type="text/javascript"  src="./js/notes.js"></script>
+<<<<<<< HEAD
```
Comment (pr656d) [inline templates/views/notes.hbs]: Issue while merging. This should not be here.

File: templates/views/notes.hbs [hunk 3]
```diff
@@ -32,5 +32,9 @@
...
+</body>   
+=======
```
Comment (pr656d) [inline templates/views/notes.hbs]: Issue while merging. This should not be here.

File: templates/views/notes.hbs [hunk 4]
```diff
@@ -32,5 +32,9 @@
...
 </body>    
+>>>>>>> main
```
Comment (pr656d) [inline templates/views/notes.hbs]: Issue while merging. This should not be here.

File: public/js/notes.js [hunk 5]
```diff
@@ -20,3 +30,11 @@ function update() {
 }
 update();
 
+function deleted(itemIndex) {
```
Comment (pr656d) [inline public/js/notes.js]: Function name not clear what function do.

File: public/js/dist/notes.dev.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+      str += "\n    <tr>\n    <th scope=\"row\">".concat(index + 1, "</th>\n    <td>").concat(element[0], "</td>\n    <td>").concat(element[1], "</td>\n    <td><button id=\"dBtn\" onclick=\"deleted(").concat(index, ");\">Delete</button>  <button id=\"eBtn\">Edit</button></td>\n    </tr>");
+    });
+  }
+
+  tableBody.innerHTML = str;
+}
+
+update();
+
+function deleted(itemIndex) {
```
Comment (pr656d) [inline public/js/dist/notes.dev.js:22]: Function name not clear what function do.

File: public/js/dist/notes.dev.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  console.log("button clicked", itemIndex);
+  itemsJson = JSON.parse(localStorage.getItem('itemsJson')); //Delete 
+
```
Comment (pr656d) [inline public/js/dist/notes.dev.js:25]: Redundant extra space

File: public/js/dist/notes.dev.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+var tableBody = document.getElementById('tableBody');
+
+function update() {
+  var str = "";
+
```
Comment (pr656d) [inline public/js/dist/notes.dev.js:7]: Redundant extra space

File: public/js/dist/notes.dev.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+  if (localStorage.getItem('itemsJson') != null) {
+    itemsJson = JSON.parse(localStorage.getItem('itemsJson'));
+    itemsJson.forEach(function (element, index) {
+      str += "\n    <tr>\n    <th scope=\"row\">".concat(index + 1, "</th>\n    <td>").concat(element[0], "</td>\n    <td>").concat(element[1], "</td>\n    <td><button id=\"dBtn\" onclick=\"deleted(").concat(index, ");\">Delete</button>  <button id=\"eBtn\">Edit</button></td>\n    </tr>");
+    });
+  }
+
```
Comment (pr656d) [inline public/js/dist/notes.dev.js:16]: Redundant extra space

File: public/js/notes.js [hunk 10]
```diff
@@ -4,6 +4,16 @@ function update() {
     let str = "";
     if (localStorage.getItem('itemsJson') != null) {
         itemsJson = JSON.parse(localStorage.getItem('itemsJson'));
+
```
Comment (pr656d) [inline public/js/notes.js]: Redundant extra space

File: public/js/notes.js [hunk 11]
```diff
@@ -4,6 +4,16 @@ function update() {
     let str = "";
```
Comment (pr656d) [inline public/js/notes.js]: 4 space indentation here. Other files containing 2 space indentations. Keep indent consistant.

**Review by pr656d — CHANGES_REQUESTED** [review]

File: public/js/notes.js (hunk 5 above)
Comment (pr656d) [reply public/js/notes.js]: "delete" would be good function name here. Avoid using past and future tens words.

File: public/js/dist/notes.dev.js (hunk 6 above)
Comment (pr656d) [reply public/js/dist/notes.dev.js:22]: "delete" would be good function name here. Avoid using past and future tens words.

**Review by pr656d — CHANGES_REQUESTED** [review]

File: public/js/notes.js (hunk 5 above)
Comment (jagruti-gothwal) [reply public/js/notes.js]: delete is reserved keywords that's why I have used deleted

Comment (pr656d) [reply public/js/notes.js]: You can use "deleteNote"

File: public/js/dist/notes.dev.js (hunk 7 above)
Comment (jagruti-gothwal) [reply public/js/dist/notes.dev.js:25]: This is auto-generated file.

File: public/js/notes.js (hunk 5 above)
Comment (pr656d) [reply public/js/notes.js]: Also "updateNote" instead of "update"

Comment (jagruti-gothwal) [reply public/js/notes.js]: okay

**Review by pr656d — APPROVED** [review]

**Pull request merged**